docs/development-guide.md

```diff
@@ -89,3 +89,10 @@ need to be met before it will be merged:
 
 When the criteria are met, a project maintainer can merge your changes into
 the project's master branch.
+
+### Backporting a PR to release branch
+
+If the PR needs to be backport to a release branch, add
+`backport-to-release-vX.Y.Z` label to the PR. Mergify bot will take care of
```

Comment:
How can contributors add the label? I can not do so...

Reply:
updated doc, once maintainers can add  labels